parcels/field.py

```diff
@@ -190,6 +190,11 @@ def __init__(self, name, data, lon=None, lat=None, depth=None, time=None, grid=N
         # self.lon, self.lat, self.depth and self.time are not used anymore in parcels.
         # self.grid should be used instead.
         # Those variables are still defined for backwards compatibility with users codes.
+        if np.all(self.grid.lat[:-1] > self.grid.lat[1:]):
```

Comment:
Does this also work for multidimensional grids, such as CurviLinear Grids? @delandmeterp, can you have a quick look at whether that would work?